tests/exercise_instructions.rs

```diff
@@ -376,41 +376,42 @@ fn fuzz_alu() {
                     l2:",
                     ),
                     &mut prng,
-                    23,
+                    Some(2),
                 );
             }
 
             imm &= 63;
 
-            test_ins(false, format!("lsh64 r{src}, {imm}"), &mut prng, 21);
-            test_ins(false, format!("rsh64 r{src}, {imm}"), &mut prng, 21);
-            test_ins(false, format!("arsh64 r{src}, {imm}"), &mut prng, 21);
-            test_ins(false, format!("hor64 r{src}, {imm}"), &mut prng, 21);
+            test_ins(false, format!("lsh64 r{src}, {imm}"), &mut prng, None);
+            test_ins(false, format!("rsh64 r{src}, {imm}"), &mut prng, None);
+            test_ins(false, format!("arsh64 r{src}, {imm}"), &mut prng, None);
+            test_ins(false, format!("hor64 r{src}, {imm}"), &mut prng, None);
 
             imm &= 31;
 
-            test_ins(false, format!("lsh32 r{src}, {imm}"), &mut prng, 21);
-            test_ins(false, format!("rsh32 r{src}, {imm}"), &mut prng, 21);
-            test_ins(false, format!("arsh32 r{src}, {imm}"), &mut prng, 21);
+            test_ins(false, format!("lsh32 r{src}, {imm}"), &mut prng, None);
+            test_ins(false, format!("rsh32 r{src}, {imm}"), &mut prng, None);
+            test_ins(false, format!("arsh32 r{src}, {imm}"), &mut prng, None);
 
-            test_ins(false, format!("be64 r{src}"), &mut prng, 21);
-            test_ins(false, format!("be32 r{src}"), &mut prng, 21);
-            test_ins(false, format!("be16 r{src}"), &mut prng, 21);
+            test_ins(false, format!("be64 r{src}"), &mut prng, None);
+            test_ins(false, format!("be32 r{src}"), &mut prng, None);
+            test_ins(false, format!("be16 r{src}"), &mut prng, None);
 
-            test_ins(true, format!("le64 r{src}"), &mut prng, 21);
-            test_ins(true, format!("le32 r{src}"), &mut prng, 21);
-            test_ins(true, format!("le16 r{src}"), &mut prng, 21);
+            test_ins(true, format!("le64 r{src}"), &mut prng, None);
+            test_ins(true, format!("le32 r{src}"), &mut prng, None);
+            test_ins(true, format!("le16 r{src}"), &mut prng, None);
         }
     }
 }
 
-fn test_ins(v0: bool, ins: String, prng: &mut SmallRng, cu: u64) {
+fn test_ins(v0: bool, ins: String, prng: &mut SmallRng, cu: Option<u64>) {
```

Comment:
Have we stopped testing CUs?

Reply:
No we do still check the CU meter at the end, but in most cases you can calculate the CUs it should take by taking the number of instructions in the assembler. No need to manually specify them everywhere.